src/audio/src/coef/src_lite_ipc4_int32_table.h

```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ *
+ */
+
+/** \cond GENERATED_BY_TOOLS_TUNE_SRC */
+
+#ifndef __SOF_AUDIO_COEFFICIENTS_SRC_src_IPC4_INT32_TABLE_H__
+#define __SOF_AUDIO_COEFFICIENTS_SRC_src_IPC4_INT32_TABLE_H__
+
+/* SRC conversions */
+#include "src_ipc4_int32_1_2_4535_5000.h"
+#include "src_ipc4_int32_10_21_3455_5000.h"
+#include "src_ipc4_int32_1_3_4535_5000.h"
+#include "src_ipc4_int32_3_2_4535_5000.h"
+#include "src_ipc4_int32_8_7_4535_5000.h"
+#include "src_ipc4_int32_16_21_4535_5000.h"
+#include "src_ipc4_int32_20_21_4167_5000.h"
+#include "../src.h"
+#include <stdint.h>
+
+/* SRC table */
+int32_t src_fir_one = 1073741824;
```

Comment:
please comment this with Q format, /* 1 << 30, with Q 2.30 format */

Reply:
Could do it with `Q_CONVERT_FLOAT(1, 30)`.

Reply:
Done